examples/build.gradle

```diff
@@ -23,9 +24,10 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:24.2.1'
-    compile 'com.android.support.constraint:constraint-layout:1.0.2'
-    compile 'com.transloadit.android.sdk:transloadit-android:0.0.2'
-    testCompile 'junit:junit:4.12'
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'androidx.appcompat:appcompat:1.0.0'
+    implementation 'com.android.support.constraint:constraint-layout:2.0.4'
+    implementation 'com.transloadit.sdk:transloadit:0.1.6'
+    implementation project(path: ':transloadit-android')
```

Comment:
```
implementation 'com.transloadit.sdk:transloadit:0.1.6'
implementation project(path: ':transloadit-android')
```

IIUC, this includes the Android SDK locally and from the repository. Is this duplication wanted?

Reply:
Kind of, as it is not the same repo. But I got rid of it now :)